great_expectations/rule_based_profiler/types/parameter_container.py

```diff
@@ -18,16 +19,34 @@
 from great_expectations.rule_based_profiler.types import Domain
 from great_expectations.types import SerializableDictDot, SerializableDotDict
 
+FULLY_QUALIFIED_PARAMETER_NAME_DELIMITER_CHARACTER: str = "$"
+
 FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER: str = "."
 
 DOMAIN_KWARGS_PARAMETER_NAME: str = "domain_kwargs"
-DOMAIN_KWARGS_PARAMETER_FULLY_QUALIFIED_NAME: str = f"$domain{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}{DOMAIN_KWARGS_PARAMETER_NAME}"
-VARIABLES_KEY: str = f"$variables{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}"
+DOMAIN_KWARGS_PARAMETER_FULLY_QUALIFIED_NAME: str = f"{FULLY_QUALIFIED_PARAMETER_NAME_DELIMITER_CHARACTER}domain{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}{DOMAIN_KWARGS_PARAMETER_NAME}"
```

Comment:
Love how these are organized, but also have a question as to how the rest of the team feels about readability. I think it's decision-call, and would love to hear from other MarioPod members.